### DISCUSSION TITLE
Julia language in Code Server

### DISCUSSION THREAD

**Discussion by anikaly10** [coder/code-server#4471] [Q&A] [answered]
Hello, I'm using code server using Google cloud shell on my iPad is there any way to run Julia program files in Code Server. I tried installing Julia Language Support extension but I'm getting an error in language environment path asking to change it in some configuration. I'm somewhat noob in this field hope you understand my question

**Comment by lg-nineoneninethreenine** [comment, marked as answer]
Hmmm... I don't think Julia support was added yet, but this is a _good_ idea to add! @jsjoeio how about adding Julia support?

Edit: Answer: https://github.com/cdr/code-server/discussions/4471#discussioncomment-1621675

**Reply by jsjoeio** [reply]
Hmm...I'm not sure. I haven't used Julia, but I would install the extension from [here](https://open-vsx.org/) and see if that works! If it doesn't, let us know!

**Reply by benz0li** [reply]
@anikaly10 code-server works fine with the Julia extension.

1.   Install [Julia](https://julialang.org/downloads/).
    *  Set/Export environment variable `JULIA_PATH` to where Julia is installed.
    *  Add `$JULIA_PATH/bin` to the `PATH` environment variable. (1)
2.  Install the [Julia extension](https://open-vsx.org/extension/julialang/language-julia).

Use version 1.4.0 of the extension with code-server 3.12.0
Use version 1.2.5 of the extension with code-server 3.10.2

---

(1) See https://github.com/docker-library/julia/blob/master/1.6/buster/Dockerfile about installing Julia on Debian and environment variables.

**Reply by anikaly10** [reply]
> @anikaly10 code-server works fine with the Julia extension.
> 
> 
> 
> 1.   Install [Julia](https://julialang.org/downloads/).
> 
>     *  Set/Export environment variable `JULIA_PATH` to where Julia is installed.
> 
>     *  Add `$JULIA_PATH/bin` to the `PATH` environment variable. (1)
> 
> 2.  Install the [Julia extension](https://open-vsx.org/extension/julialang/language-julia).
> 
> 
> 
> Use version 1.4.0 of the extension with code-server 3.12.0
> 
> Use version 1.2.5 of the extension with code-server 3.10.2
> 
> 
> 
> ---
> 
> 
> 
> (1) See https://github.com/docker-library/julia/blob/master/1.6/buster/Dockerfile about installing Julia on Debian and environment variables.

Hey thanks, it's working..
